tests/test_analysis/test_feature_selection/test_mrmr.py

```diff
@@ -58,29 +58,39 @@ def df_with_regressors() -> Dict[str, pd.DataFrame]:
     }
 
 
+@pytest.mark.parametrize("fast_redundancy", [True, False])
```

Comment:
I think you should also update inference tests for MRMR.

Reply:
There are no changes for tests of the transform itself. Is it ok?

Reply:
We can add the same parametrization as in mrmr test, don't know whether it is usefull

Reply:
I think we should try to do it. At least for some tests.